jobserver/models/user.py

```diff
@@ -48,6 +48,17 @@ def order_by_name(self):
 class UserManager(DjangoUserManager.from_queryset(UserQuerySet)):
     use_in_migrations = True
 
+    def create_superuser(self, email, password, **extra_fields):
+        username = email
+        return super().create_superuser(
+            username,
+            email,
+            password,
+            is_staff=True,
+            is_superuser=True,
+            roles=[CoreDeveloper],
+        )
```

Comment:
Django sets `is_staff` and `is_superuser` to `True` by default and raises a `ValueError` if either are `False`, so we need not set them here.[^1] Whilst setting them here makes no difference, functionally, it did lead me to ask "Why are we setting them here?"

```suggestion
    def create_superuser(self, email, password, **extra_fields):
        username = email
        return super().create_superuser(
            username, email, password, roles=[CoreDeveloper]
        )
```

[^1]: https://github.com/django/django/blob/428d06ccef09e70bcef9869c5a9404863b2fc7d8/django/contrib/auth/models.py#L163-L172

Reply:
That seems obvious now you mention it!